telegram/user.py

```diff
@@ -206,6 +206,19 @@ def send_photo(self, *args, **kwargs):
         """
         return self.bot.send_photo(self.id, *args, **kwargs)
 
+    def send_media_group(self, *args, **kwargs):
+        """Shortcut for::
+
+            bot.send_media_group(User.id, *args, **kwargs)
+
+        Where User is the current instance.
```

Comment:
Same as with chat, here `update.message.from_user.id` is used

Reply:
Same as above